src/virtac/pv.py

```diff
@@ -378,41 +383,40 @@ def _setup_pv_monitoring(self, pv_names, callbacks):
 
         Args:
             pv_names (list[str]): A list of PV names to monitor using channel access.
-            callbacks (list[Callable]): A list of functions to execute when the
+            callbacks (list[CallbackType]): A list of functions to execute when the
                 associated PV changes value.
         """
         if callbacks is None:
             callbacks = [self._callback]
 
         for pv_name in pv_names:
-            if not isinstance(pv_name, str):
```

Comment:
Same issue with removing type checks. Are you expecting the type checker to prevent this class of problem from occurring?